test/distributed/elastic/launchers/__init__.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
```

Comment:
Would I be correct if I assume, there will be followup efforts to consolidate with `https://github.com/pytorch/pytorch/blob/master/torch/distributed/launch.py`?